checkins.json

```diff
@@ -1,47 +1,82 @@
 {
```

Comment:
in order to be consistent with other acqusitions schemas, please use underscore_notation instead of camelCase for field/property names.

Reply:
Also, this schema should be renamed to `checkin_collection`